stream-compositions/cursors/transaction-cursor/src/main/resources/application-local.yml

```diff
@@ -50,7 +50,7 @@ eureka:
       role: live
   client:
     serviceUrl:
-      defaultZone: http://localhost:8080/registry/eureka/
+      defaultZone: http://localhost:8761/eureka/
```

Comment:
This runs on 8080 for everyone running the service using docker.

Reply:
@shafiquech I don't think so. If we look at all the composition services, they have the local config set similarly. txn-cursor was the only one out of sync.

![image](https://github.com/Backbase/stream-services/assets/80056480/4bb85add-8f74-4227-870b-212f7e65dc4d)

Reply:
Maybe they used 8080 earlier... but when we use the local setup using `local-backend-setup` repo below, the port is 8761:
https://github.com/backbase/local-backend-setup/tree/main/development/docker-compose#set-up-backbase-local-environment